include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3933,9 +3937,6 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
         dof_indices[v] = numbers::invalid_unsigned_int;
```

Comment:
We can probably just skip the `else` branch then.

Reply:
Correct. Updated.